app/controllers/collections_controller.rb

```diff
@@ -0,0 +1,64 @@
+##
+# Controller used for a user's collection
+class CollectionsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :set_collection, only: [:show, :edit, :update, :destroy]
+  load_and_authorize_resource
+
+  # GET /collections
+  def index
+    @collections = Collection.where(user: current_user)
```

Comment:
Can we use `accessible_by` here?  I can't quite remember if that's the use-case for this or not.